gazebo_ros/src/gazebo_ros_init.cpp

```diff
@@ -66,20 +66,12 @@ void GazeboRosInit::Load(int argc, char ** argv)
     "/clock",
     rclcpp::QoS(rclcpp::KeepLast(10)).transient_local());
 
-  // Get publish rate from parameter if set
-  rclcpp::Parameter rate_param;
-  if (impl_->ros_node_->get_parameter("publish_rate", rate_param)) {
-    if (rclcpp::ParameterType::PARAMETER_DOUBLE == rate_param.get_type()) {
-      impl_->throttler_ = Throttler(rate_param.as_double());
-    } else if (rclcpp::ParameterType::PARAMETER_INTEGER == rate_param.get_type()) {
-      impl_->throttler_ = Throttler(rate_param.as_int());
-    } else {
-      RCLCPP_WARN(impl_->ros_node_->get_logger(),
-        "Could not read value of param publish_rate [%s] as double/int, using default %ghz.",
-        rate_param.value_to_string().c_str(),
-        GazeboRosInitPrivate::DEFAULT_PUBLISH_FREQUENCY);
-    }
-  }
+  // Publish rate parameter
```

Comment:
The instructions [here](https://github.com/ros-simulation/gazebo_ros_pkgs/wiki/ROS-2-Migration:-ROS-Clocks-and-sim-time#clock-rate) on how to set the publish rate should work